src/Modules/Windows-Core/Microsoft.PowerShell.Diagnostics/Diagnostics.format.ps1xml

```diff
@@ -0,0 +1,79 @@
+﻿<Configuration>
+   
+
+  <!-- ################ VIEW DEFINITIONS ################ -->
```

Comment:
These were copied from a different tree location after a comment in the first PR commit, and a subsequent conversation, from Francisco.